packages/odyssey-react-mui/src/RadioGroup.tsx

```diff
@@ -89,7 +89,17 @@ const RadioGroup = ({
     [onChangeProp]
   );
   const renderFieldComponent = useCallback(
-    ({ ariaDescribedBy, errorMessageElementId, id, labelElementId }) => (
+    ({
+      ariaDescribedBy,
+      errorMessageElementId,
+      id,
+      labelElementId,
+    }: {
+      ariaDescribedBy?: string;
+      errorMessageElementId?: string;
+      id: string;
+      labelElementId: string;
+    }) => (
```

Comment:
Can we `Pick<FieldComponentProps, "">` these?

Reply:
Yea, I'll address this here and in other places, but the gist of it is that where we define this, we don't actually have a named type for these properties to reference.